<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPreferencesWindow</class>
 <widget class="QMainWindow" name="QPreferencesWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cue TX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/trayicon.png</normaloff>:/trayicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBoxOutputSettings">
      <property name="title">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBoxInputSettings">
      <property name="title">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelInput">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelOutput">
      <property name="minimumSize">
       <size>
        <width>38</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <property name="topMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacerButtons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBoxInput">
      <item>
       <property name="text">
        <string>MSC (MIDI)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MSC (GrandMA 2 via Ethernet)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OSC (ETC Eos)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OSC (Custom)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBoxOutput">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>None (Log only)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MSC (MIDI)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MSC (GrandMA2 via Ethernet)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OSC (disguise/d3)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OSC (ETC Eos)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OSC (QLab)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OSC (Custom)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QPreferencesWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
